pre_award/assess/assessments/routes.py

```diff
@@ -1311,25 +1311,30 @@ def request_changes(application_id, sub_criteria_id, theme_id):
     assessment_status = determine_assessment_status(sub_criteria.workflow_status, state.is_qa_complete)
     theme_answers_response = get_sub_criteria_theme_answers_all(application_id, theme_id)
 
-    form = RequestChangesForm(
-        question_choices=[(question["field_id"], question["question"]) for question in theme_answers_response]
-    )
+    question_choices = [
+        (question["field_id"], question["question"], question.get("answer", "")) for question in theme_answers_response
+    ]
```

Comment:
We only really need `field_ids` to pass to the Form class. I would change this to be just an array of field_ids. We should only pass information we need to the class/function.

You are using the `question` in the Form class for the textarea description, we can rewrite that to be "Reason for {field_id}" or just "Reason".